### PR TITLE
Update batch_actions.rst

### PR DESCRIPTION
**Pull request by rccc**
* It seems that the "targetId" parameter is no more used. Fot batch actions i get the "idx" parameter instead.
* The addFlash method is deprecated

### CODE REVIEW

File: Resources/doc/reference/batch_actions.rst [hunk 1]
```diff
@@ -189,13 +189,12 @@ granularity), the passed query is ``null``.
...
-        $request = $this->get('request');
         $modelManager = $this->admin->getModelManager();
 
-        $target = $modelManager->find($this->admin->getClass(), $request->get('targetId'));
+        $idx = $this->get('request')->get('idx');
 
-        if( $target === null){
-            $this->addFlash('sonata_flash_info', 'flash_batch_merge_no_target');
+        if(empty($idx){
+            $this->getFlashBag()->add('sonata_flash_info', 'flash_batch_merge_no_target');
```
Comment (Soullivaneuh) [inline Resources/doc/reference/batch_actions.rst]: `$this->addFlash` instead.

See: https://github.com/sonata-project/SonataAdminBundle/blob/2bb5c41dd77f73f8b449021278dc23dc0826e76e/Controller/CRUDController.php#L1225

File: Resources/doc/reference/batch_actions.rst [hunk 2]
```diff
@@ -213,14 +212,14 @@ granularity), the passed query is ``null``.
 
             $modelManager->update($selectedModel);
         } catch (\Exception $e) {
-            $this->addFlash('sonata_flash_error', 'flash_batch_merge_error');
+            $this->getFlashBag()->add('sonata_flash_error', 'flash_batch_merge_error');
```
Comment (Soullivaneuh) [inline Resources/doc/reference/batch_actions.rst]: Please revert it.

File: Resources/doc/reference/batch_actions.rst [hunk 3]
```diff
@@ -213,14 +212,14 @@ granularity), the passed query is ``null``.
...
 
             return new RedirectResponse(
               $this->admin->generateUrl('list',$this->admin->getFilterParameters())
             );
         }
 
-        $this->addFlash('sonata_flash_success', 'flash_batch_merge_success');
+        $this->getFlashBag()->add('sonata_flash_success', 'flash_batch_merge_success');
```
Comment (Soullivaneuh) [inline Resources/doc/reference/batch_actions.rst]: Please revert it.

File: Resources/doc/reference/batch_actions.rst (hunk 2 above)
Comment (rccc) [inline Resources/doc/reference/batch_actions.rst]: You are right. I was thinking it was the old symfony method  ...

**Comment by rccc** [comment]
Is it ok now ?

**Comment by Soullivaneuh** [comment]
@rccc Well, you closed the PR and deleted your branch so... not really. :-)

**Comment by rccc** [comment]
I didn't find the revert button. 

i won't touch this f... doc anymore.

**Pull request closed without merging**